sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/AddPartitionExec.scala

```diff
@@ -27,7 +27,7 @@ import org.apache.spark.sql.connector.catalog.{SupportsAtomicPartitionManagement
 /**
  * Physical plan node for adding partitions of table.
  */
-case class AlterTableAddPartitionExec(
+case class AddPartitionExec(
```

Comment:
Shall we rename the logical plan as well? The logical plan for REAME is `RenameTable` as well.

Reply:
Here is the PR https://github.com/apache/spark/pull/31596 @cloud-fan @imback82 @dongjoon-hyun Could you take a look at it.